ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -23,36 +23,51 @@ images:
   inputs:
     bin:
       as:
-      - builder
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
       paths: null
   to: cli
 - dockerfile_path: images/cli/Dockerfile.rhel
   from: tests-base
   inputs:
     bin:
       as:
-      - builder
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
       paths: null
   to: tests
 - dockerfile_path: images/cli/Dockerfile.rhel
   from: must-gather-base
   inputs:
     bin:
       as:
-      - builder
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
       paths: null
   to: must-gather
 - context_dir: images/deployer/
   dockerfile_path: Dockerfile.rhel
   from: cli
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      paths: null
   to: deployer
 - context_dir: .
   dockerfile_path: images/cli-artifacts/Dockerfile.rhel
   from: cli
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      paths: null
   to: cli-artifacts
 - context_dir: images/recycler/
   dockerfile_path: Dockerfile.rhel
   from: cli
+  inputs:
+    bin:
+      as:
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
```

Comment:
recycler doesn't need this as it is build just on the cli image, but it is safer if copied around and doesn't hurt